library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -474,11 +480,18 @@ return_status libspdm_build_response(IN void *context, IN uint32_t *session_id,
                     spdm_context,
                     spdm_context->latest_session_id,
                     LIBSPDM_SESSION_STATE_ESTABLISHED);
+                session_state_established = TRUE;
             }
             break;
         }
     }
 
+    if (session_state_established == TRUE) {
```

Comment:
Please done compare a boolean value. It is meaningless.

Just use if (session_state_established)